src/lightbox/LightboxActionSheet.js

```diff
@@ -50,7 +50,7 @@ const shareImageDirectly = ({ src, auth }: DownloadImageType) => {
 };
 
 const actionSheetButtons: ButtonType[] = [
-  { title: 'Download file', onPress: downloadImage },
+  { title: 'Download image', onPress: tryToDownloadImage },
```

Comment:
Make sure to update the `messages_en.json` file.

Reply:
Will update!

I wonder if there's any way that this caught via the test suite.